docs/user/metal/install_ipi.md

```diff
@@ -52,11 +52,13 @@ accessible over the external network which may not be desirable.
   * A private network used for PXE based provisioning.
   * You must specify `provisioningNetworkInterface` to indicate which
     interface is connected to this network on the control plane nodes.
-  * The provisioning network may be "Managed" (default), "Unmanaged," or
+  * The `provisioningNetwork` may be "Managed" (default), "Unmanaged," or
     "Disabled."
-  * In managed mode, DHCP and TFTP are configured to run in the cluster. In
+  * In `Managed` mode, DHCP and TFTP are configured to run in the cluster. In
     unmanaged mode, TFTP is still available but you must configure DHCP
     externally.
+  * In `Disabled` mode, no second NIC is required, but an additional controlplane
+    IP must be specified by the `clusterProvisioningIP`.
```

Comment:
I wonder should we break this whole section out of the network requirements part, and describe the provisioningNetwork options/scenarios in more detail?

Reply:
+1

We need a clearer way to present the choices on provisioning. There's a lot of possible configurations now, so complexity is quite a bit higher.

Reply:
Yes, a separate section would make sense.